scripts/install-ginkgo.sh

```diff
@@ -1,10 +1,21 @@
 #!/usr/bin/env bash
 
-GOPATH="/root/go"
+REQUIRED_GINKGO_VERSION="2.9.2"
```

Comment:
Required version should get brought in from the go.mod:

`$(awk '/ginkgo/ {printf "%s/ginkgo@%s", $1, $2}' go.mod)`